site2/docs/admin-api-namespaces.md

```diff
@@ -905,7 +915,7 @@ admin.namespaces().removeDeduplicationSnapshotInterval(namespace)
 
 ### Namespace isolation
 
-Coming soon.
+You can use the Pulsar isolation policy to allocate resources (broker and bookie) for a namespace. For more information, see [here](administration-isolation.md).
```

Comment:
I'd just linkify the term "Pulsar isolation policy" and remove the entire second sentence.

Reply:
OK. Done.